assembl/static2/js/app/components/debate/common/post/postBody.jsx

```diff
@@ -38,18 +40,23 @@ const ExtractInPost = ({ id, children }: ExtractInPostProps) => (
   </span>
 );
 
-const postBodyReplacementComponents = {
-  iframe: (attributes) => {
-    const { src } = attributes;
-    const regexpMatch = src.match(youtubeRegexp);
-    if (regexpMatch) {
-      const videoId = regexpMatch[1];
-      return <YoutubeTheater videoId={videoId} />;
-    }
-    return <iframe title="post-embed" {...attributes} />;
+const postBodyReplacementComponents = afterLoad => ({
+  iframe: attributes => (
+    // the src iframe url is different from the resource url
+    <Embed url={attributes['data-source-url'] || attributes.src} defaultEmbed={<iframe title="post-embed" {...attributes} />} />
+  ),
+  a: (attributes) => {
+    const embeddedUrl = isSpecialURL(attributes.href);
+    const origin = (
+      <a href={attributes.href} className="linkified" target="_blank">
+        {attributes.href}
+      </a>
+    );
+    if (embeddedUrl) return origin;
+    return [origin, <URLMetadataLoader url={attributes.href} afterLoad={afterLoad} />];
```

Comment:
Don't you have warnings here because a key is missing for each item in the array?